ccmlib/scylla_repository.py

```diff
@@ -48,7 +48,7 @@ def setup(version, verbose=True):
 
         url = os.environ.get('SCYLLA_CORE_PACKAGE', os.path.join(
             s3_url, 'scylla-package.tar.gz'))
-        download_version(version, verbose=verbose, url=url, target_dir=os.path.join(
+        core_version = download_version(version, verbose=verbose, url=url, target_dir=os.path.join(
```

Comment:
why are you adding this ? I think the version is already part of the cdir, why adding it again ?

Reply:
o.k, now I get it, should be named `package_version` like where it's used